packages/fxa-auth-client/.eslintrc.json

```diff
@@ -0,0 +1,10 @@
+{
+  "extends": [
+    "react-app",
```

Comment:
Ah. I probably just saw that it had .ts files and figured react-app was the quickest way to set the correct parser+rules.

I'll see if I can find another package that lints TypeScript that doesn't have React since I'm not sure how well our "plugin:fxa/client" or "plugin:fxa/server" or "plugin:fxa/recommended" configs support TypeScript.

I was also hoping we could kill/refactor that old [eslint-plugin-fxa](https://github.com/mozilla/eslint-plugin-fxa/tree/master/lib/configs) and possibly [move it into this monorepo](https://github.com/mozilla/fxa/issues/13051) to easily share linting configs between projects without needing to do a npm publish (and then expand the configs to include "plugin:fxa/typescript", "plugin:fxa/react", "plugin:fxa/jest", "plugin:fxa/nestjs", "plugin:fxa/storybook", etc).